src/main/java/edu/pdx/cs/ConnectionInfo.java

```diff
@@ -0,0 +1,69 @@
+package edu.pdx.cs;
+
+import java.nio.file.Path;
+
+/**
+ * This class is represents a <code>ConnectionInfo</code>
+ * This class has necessary field to create an FTP connection
+ * using getter methods.
+ */
+public class ConnectionInfo {
+
+    private Path currentDir;
+    private String IP;
```

Comment:
Current directory is a good fit for another part of the program. We can probably hold off on implementing it for now.

Can you rename `IP` to something like `host`? To reflect that we may be using IP or URL.

port can be an int. Check out the signature of this method in SocketClient.
`public void connect(String hostname, int port)`
https://commons.apache.org/proper/commons-net/apidocs/org/apache/commons/net/SocketClient.html